frontend/javascripts/oxalis/model/reducers/skeletontracing_reducer.js

```diff
@@ -370,6 +370,31 @@ function SkeletonTracingReducer(state: OxalisState, action: Action): OxalisState
             })
             .getOrElse(state);
         }
+        case "SET_ACTIVE_TREE_BY_NAME": {
+          const { treeName } = action;
+          const { trees } = skeletonTracing;
+          let treeWithMatchingName = null;
+          Object.keys(trees).forEach(treeId => {
+            const treeIdAsNumber = ((treeId: any): number);
+            if (trees[treeIdAsNumber].name === treeName) {
+              treeWithMatchingName = trees[treeIdAsNumber];
+            }
+          });
```

Comment:
I think, this block can be shortened to:
```
const treeWithMatchingName = Object.values(trees).find(tree => tree.name == treeName)
```

(in case, flow doesn't get the type information `object.values()` might be replacable with `_.values`)

Reply:
Thanks for this feedback/nagging about this :+1: